docs/src/content/docs/id/guides/sidebar.mdx

```diff
@@ -216,45 +269,44 @@ Sebuah grup yang dihasilkan secara otomatis termasuk halaman dengan frontmatter
 		{
 			label: 'Panduan',
 			items: [
-				{ label: 'Sebuah halaman', link: '#' },
+				{ label: 'Sebuah halaman', link: '' },
 				{
 					label: 'Label sidebar custom',
-					link: '#',
+					link: '',
 					badge: { text: 'Baru', variant: 'tip' },
 				},
-				{ label: 'Halaman lain', link: '#' },
+				{ label: 'Halaman lain', link: '' },
 			],
 		},
 	]}
 />
 
-:::note
-Konfigurasi frontmatter `sidebar` hanya digunakan untuk tautan yang dihasilkan secara otomatis dan akan diabaikan untuk tautan yang ditentukan secara manual.
+:::catatan
```

Comment:
```suggestion
:::note
```

The aside type should not be translated.

Reply:
i see, i will update it soon.